src/datachain/data_storage/metastore.py

```diff
@@ -167,32 +167,17 @@ def mark_storage_indexed(
         This method should be called when index operation is finished.
         """
 
-    @abstractmethod
-    def mark_storage_not_indexed(self, uri: StorageURI) -> None:
-        """
-        Mark storage as not indexed.
-        This method should be called when storage index is deleted.
-        """
-
     @abstractmethod
     def update_last_inserted_at(self, uri: Optional[StorageURI] = None) -> None:
```

Comment:
is this one still relevant?

Reply:
Yes, this is still being used in old deprecated listing from CLI. It will be removed when we refactor that last left part (CLI)